library/Icingadb/Common/Auth.php

```diff
@@ -107,20 +107,20 @@ public function isMatchedOn(string $queryString, Model $object): bool
      * fetching either hosts or services. It also applies custom variable restrictions and
      * obfuscations. (`icingadb/blacklist/variables` and `icingadb/protect/variables`)
      *
-     * @param Query $query
+     * @param Query $orgQuery
      *
      * @return void
      */
-    public function applyRestrictions(Query $query)
+    public function applyRestrictions(Query $orgQuery)
```

Comment:
Why? You don't assign anything to $query anyway.

Reply:
Yes, but there is also something like this...
```php
foreach($queries as $query) ...
```
Then `$query` is no longer the original one.

Reply:
I see. But then assign `$orgQuery` in the body, don't change the name of the parameter.